app/views/shared/_taxonomy_navigation.html.erb

```diff
@@ -41,6 +41,39 @@
       </div>
     <% end %>
 
+    <% if taxonomy_navigation[:news_and_communication].present? %>
+      <div class="taxonomy-navigation__section">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: t('supergroups.news_and_communications'),
+          heading_level: 3,
+          font_size: 24,
+          margin_bottom: 2
+        } %>
+
+        <% taxonomy_navigation[:news_and_communication][:promoted_content].in_groups_of(3, false) do |promo_group| %>
+          <div class="grid-row">
+          <% promo_group.each do |promo| %>
+            <div class="column-one-third">
+              <%= render "govuk_publishing_components/components/image_card", {
+                context: promo[:image][:context],
+                href: promo[:link][:path],
+                heading_text: promo[:link][:text],
+                image_src: promo[:image][:url],
+                image_alt: "",
```

Comment:
Ah yes, I think we talked about this in the accessibility session.  I should have been a bit more overt.  I think when the image is part of a link it should use the link text instead of the alt text?  (I may be wrong).
Either way, I can add the alt text and it shouldn't make anything worse. 👍